chrome/content/zotero/elements/itemPaneSidenav.js

```diff
@@ -28,9 +28,11 @@
 {
 	class ItemPaneSidenav extends XULElementBase {
 		content = MozXULElement.parseXULToFragment(`
-			<html:div class="inherit-flex highlight-notes-inactive">
+			<html:div class="inherit-flex highlight-notes-inactive"
+				tabindex="0" role="tab" aria-labelledby="sidenav-info-btn">
```

Comment:
I couldn't really think of a better way to name it...

"Info" seemed like a reasonable label, since clicking this is essentially equivalent to clicking on the "Info" button when the notes pane is opened in the reader. And then it shows the pane with "Item info". But we could very much rename it for something that would be more descriptive. Something like "Item details?"